Jenkinsfile

```diff
@@ -0,0 +1,76 @@
+@Library('jenkins.shared.library') _
+
+environment {
+  HELM_IMAGE = "infoblox/helm:3.2.4-5b243a2"
+  REGISTRY = "core-harbor-prod.sdp.infoblox.com"
+  TAG = sh(script: "git describe --always --long --tags", returnStdout: true).trim()
+}
+
+pipeline {
+  agent {
+    label 'ubuntu_docker_label'
+  }
+  stages {
+    stage("Build Image") {
+      sh "docker build . -t tugger:${version}"
+    }
+    stage("Push Image") {
+      when {
+        anyOf {
+          branch 'master'
+          branch 'jenkinsfile'
+        }
+      }
+      steps {
+        script {
+          signDockerImage('tugger', env.TAG, 'infoblox')
+        }
+      }
+    }
+    stage("Package Chart") {
+      dir("chart") {
+        sh '''
+          sed -i "s!repository: .*!repository: $REGISTRY/infoblox/tugger!g" tugger/values.yaml
+          sed -i "s!tag: .*!tag: $TAG!g" tugger/values.yaml
+        '''
+        withAWS(credentials: "CICD_HELM", region: "us-east-1") {
+          sh '''
+            docker run --rm \
+                -v $(pwd):/pkg \
+                $HELM_IMAGE package /pkg/tugger --version $TAG -d /pkg
+            docker run --rm \
+                -e AWS_REGION \
+                -e AWS_ACCESS_KEY_ID \
+                -e AWS_SECRET_ACCESS_KEY \
+                -v $WORKSPACE/$DIRECTORY/helm:/pkg \
+                $HELM_IMAGE s3 push /pkg/tugger infobloxcto
+            cd ..
+            mkdir build
```

Comment:
`mkdir -p build` just in case of rebuilding

Reply:
Oops, this whole section was supposed to be removed.